src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Helpers/Emitter/ConfigurationBinder.cs

```diff
@@ -25,13 +25,13 @@ private void EmitBinder_ConfigurationBinder()
                 }
 
                 _emitBlankLineBeforeNextStatement = false;
-                EmitRootBindingClassBlockStart(Identifier.GeneratedConfigurationBinder);
+                EmitRootBindingClassStartScope(Identifier.GeneratedConfigurationBinder);
 
                 EmitGetMethods();
                 EmitGetValueMethods();
                 EmitBindMethods_ConfigurationBinder();
 
-                _writer.WriteBlockEnd();
+                EmitEndScope();
```

Comment:
Nit: blocks define a new scope but not all scopes are defined using blocks. Perhaps use "block" to emphasize that all the method is doing is emit C-style block syntax.